### PR TITLE
Clean stop timetable edition

### PR DESCRIPTION
**Pull request by vpassama**
Because Mtt doesn't provide anymore the function 'edit a stopPoint', some cleaning is needed.

### CODE REVIEW

**Comment by vpassama** [comment]
Hi,

I have some problems with the FrequencyControllerTest result : 

```
1) CanalTP\MttBundle\Tests\Functional\Controller\FrequencyControllerTest::testEditForm
Failed asserting that false is true.
```

If i check the html content : 

```
public function testEditForm() {
    echo $crawler->text()
}
```

I see that the result is returning an error :

```
Navitia\Component\Exception\NotFound\UnknownObjectException: Navitia error message: The region fr-cen doesn't exists
    at n/a
        in /var/www/nmm-merge/vendor/canaltp/navitia/NavitiaExceptionFactory.php line 44

    at Navitia\Component\NavitiaExceptionFactory->create('404', 'unknown_object', 'The region fr-cen doesn't exists')
        in /var/www/nmm-merge/vendor/canaltp/navitia/Service/NavitiaService.php line 231

    at Navitia\Component\Service\NavitiaService->errorProcessor('{"message":"The region fr-cen doesn't exists","error":{"message":"The region fr-cen doesn't exists","id":"unknown_object"}}', '404')
        in /var/www/nmm-merge/vendor/canaltp/navitia/Service/NavitiaService.php line 172

    at Navitia\Component\Service\NavitiaService->callApi(object(CoverageRequest), null)
        in /var/www/nmm-merge/vendor/canaltp/navitia/Service/NavitiaService.php line 75

    at Navitia\Component\Service\NavitiaService->process(array('api' => 'coverage', 'parameters' => array('region' => 'fr-cen', 'path_filter' => 'routes/route:TTR:Nav155', 'parameters' => '?depth=3')), null, '5000', true)
```

**Comment by vpassama** [comment]
Something is broken in the left_menu when editing a stopTimetable.

There is a link to preview page but no externalStopPointId is provided. Maybe we should provide the 'first' stopPoint on the route to this menu ?

**Comment by vpassama** [comment]
I'm currently working on the pdf generation. It would be great to deploy a new 'line-timetable' branch in https://github.com/CanalTP/AmqpMttWorkers because some changes are needed.

**Comment by vpassama** [comment]
I have a failure in TimetableControllerTest.php : 

```
Starting test 'CanalTP\MttBundle\Tests\Functional\Controller\TimetableControllerTest::testAccess'.
PHP Fatal error:  Call to a member function getCustomer() on a non-object in /var/www/nmm-merge/vendor/canaltp/mtt-bundle/Controller/StopTimetableController.php on line 133
```

https://github.com/CanalTP/MttBundle/blob/line-timetable/Tests/Functional/Controller/TimetableControllerTest.php#L116-L121

I don't know if I really broke something here. When I try to access this view when I'm not logged in, I have the same error.

**Comment by dvdn** [comment]
As we spoke about AmqpMttWorkers, a new branch is available :
https://github.com/CanalTP/AmqpMttWorkers/tree/line-timetable

**Comment by vpassama** [comment]
:+1:

**Comment by vpassama** [comment]
Hey @dvdn, as said before : 

> Something is broken in the left_menu when editing a stopTimetable.
>
> There is a link to preview page but no externalStopPointId is provided. Maybe we should provide the > 'first' stopPoint on the route to this menu ?

Something is broken between edition and preview in the left_menu. Do you want me to fix this ?

File: Resources/views/Layouts/macros.html.twig [hunk 1]
```diff
@@ -103,13 +121,13 @@
 {% endmacro %}
 
 {# params string colorAltern    how to style timegrid, color direction, line or column #}
-{% macro calendar(dom_id, stopPointLevel, externalNetworkId, stopTimetable, layout, block, calendar, notes, hours_range, linesMin, notesType, colorAltern = 'line') %}
+{% macro calendar(dom_id, externalNetworkId, stopTimetable, layout, block, calendar, notes, hours_range, linesMin, notesType, colorAltern = 'line') %}
```
Comment (kun2985) [inline Resources/views/Layouts/macros.html.twig]: You deleted the parametre "stopPointLevel" there, but this parametre is not deleted in the file "/Resources/views/Layouts/timegrid.html.twig".

Comment (vpassama) [inline Resources/views/Layouts/macros.html.twig]: Right but is this file really used somewhere ?

**Comment by kun2985** [comment]
I use your template https://github.com/Tisseo/MttTemplate to test.
I found that there is some probleme in the template compared with the pr:
* stop.html.twig
  * at line 63 the macro is not exist: {{ macros.notes(notes, notesType) }}
  * at line 76 include "CanalTPMttBundle:Layouts:common.js.twig" should be "common.html.twig"
  * at line 76 include "CanalTPMttBundle:Layouts:common.html.twig" without the parameter "timetable" who is used in the file "common.html.twig"

* /mtt-bundle/Resources/public/js/layout.js
  * at line 41 "'blockType': $(this).data('type')" should be "'blockType': $(this).data('block-type')"

**Comment by vpassama** [comment]
Yes, https://github.com/Tisseo/MttTemplate is not maintained anymore because of the lack of time. You should just use the default templates provided by fixtures.

**Comment by kun2985** [comment]
I can't find a template for lines. I have defaults templates for stops in the project. What do you mean "the default templates provided by fixtures"?

**Comment by kun2985** [comment]
Can you tell me where to find the link to view "page to edit a simple timecard" for a line.

**Comment by vpassama** [comment]
This PR isn't about line timetable edition, it just clean the code from stopPoint edition functions. If you want a default template for lines, you should checkout the next PR : https://github.com/CanalTP/MttBundle/pull/90

**Comment by kun2985** [comment]
OK, there is still a error, I think:
* /mtt-bundle/Resources/public/js/layout.js
  * at line 41 "'blockType': $(this).data('type')" should be "'blockType': $(this).data('block-type')"

**Pull request merged**